dockers/docker-fpm-frr/daemons

```diff
@@ -0,0 +1,65 @@
+# This file tells the frr package which daemons to start.
+#
+# Sample configurations for these daemons can be found in
+# /usr/share/doc/frr/examples/.
+#
+# ATTENTION:
+#
+# When activation a daemon at the first time, a config file, even if it is
+# empty, has to be present *and* be owned by the user and group "frr", else
+# the daemon will not be started by /etc/init.d/frr. The permissions should
+# be u=rw,g=r,o=.
+# When using "vtysh" such a config file is also needed. It should be owned by
+# group "frrvty" and set to ug=rw,o= though. Check /etc/pam.d/frr, too.
```

Comment:
Did we set all the owners and groups accordingly to this note?

Reply:
This functionality hasn't changed from what was already there and working prior to the breakages. The owners and groups are unchanged.